_config.yml

```
@@ -1,4 +1,7 @@
-# _config.yml
 title: Ganesh Palanikumar
-description: Personal website of Ganesh Palanikumar
-theme: minima
+description: >-
+  Backend Engineer | Creator of Synapso & Areq | Building tools at the intersection of AI and engineering.
+theme: jekyll-theme-hack
+
+# Optional social links
+github_username: ganesh-palanikumar
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)